### DISCUSSION TITLE
What music genres do you like?

### DISCUSSION THREAD

**Discussion by kohei-ts** [NAIST-SE/AttractingContributors#820] [2023 Attraction Contributions]
I want to listen to a wide range of music with systematic understanding. So, I want to know what music genres you listen most often. 
You can add not only fine-grained genres (ex. Gacha Pop, Hyperpop, Alternative Rock, Progressive Rock), but also coarse-grained genres (ex. Pops, Rock, EDM).

Come to my fork:
https://github.com/kohei-ts/AttractingContributors-MusicGenre

Add to my README file.

I look forward to your contributions

**Comment by KKKKKXY** [comment]
I added my music genres into your list.(githubid: KKKKKXY)

**Comment by Sanaa3131** [comment]
I added my music genres to your list, could you please add your favorite place to visit in your country to my README file?
https://github.com/Sanaa3131/Touristic-places-wish-list/blob/main/README.md
Thank you in advance

**Reply by kohei-ts** [reply]
Thx!!
I also added to your list!

**Comment by oto10** [comment]
I added my music genres to your list.
Could you please add your favorite music to my README.md?
https://github.com/oto10/AttractingContributors-your-favorite-music
Thank you in advance.

**Reply by kohei-ts** [reply]
Thx!!
I also added your list!
